cellprofiler/gui/pipelinecontroller.py

```diff
@@ -1560,6 +1575,15 @@ def on_pathlist_file_command(self, paths, cmd):
                 self.on_pathlist_browse(
                     None,
                     default_dir=path)
+        elif cmd == self.PATHLIST_CMD_BROWSE_FOLDER:
```

Comment:
I think in this case and the case below, it might be better to have the following:
```
        elif cmd in [self.PATHLIST_CMD_BROWSE, self.PATHLIST_CMD_BROWSE_FOLDER]:
            browse_function = self.on_pathlist_browse_folder if cmd == self.PATHLIST_CMD_BROWSE_FOLDER else self.on_pathlist_browse
            if len(paths) == 0 or not paths[0].startswith("file:"):
                browse_function(None)
            else:
                path = urllib.url2pathname(paths[0][5:])
                path = os.path.split(path)[0]
                browse_function(None, default_dir=path)
```
That way we only have the `urllib` parsing occurring once and don't have repeated code.